app/views/users/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Here is the details for a given user</h1>
\ No newline at end of file
+<%= render 'shared/user', user: @user %>
+<div class="bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+  <% @user.recent_posts.each do |post| %>
+    <%= link_to user_posts_path(@user) do %>
+    <%= render 'shared/post', post: post %>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] Please add `see all posts` link at the end of the user show page, Clicking this link opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

![image](https://github.com/AlejandraKeber/BlogApp/assets/68796232/84224294-e6d7-4fe6-9636-16a066e79b35)